crimsobot/cogs/reactions.py

```diff
@@ -0,0 +1,183 @@
+import logging
+
+from discord.ext import commands
+from discord.ext.commands import has_guild_permissions
+from tortoise.exceptions import DoesNotExist
+
+from config import ADMIN_USER_IDS
+from crimsobot.bot import CrimsoBOT
+from crimsobot.models.fun_facts import FunFacts, NoFactsExist
+from crimsobot.utils import tools as c
+
+log = logging.getLogger(__name__)
+
+
+class CleanMentions(commands.Converter):
+    async def convert(self, ctx: commands.Context, string: str) -> str:
+        """Clean up those silly mention inconsistencies across platforms."""
+        for mention in ctx.message.mentions:
+            string = string.replace(f'<@!{mention.id}>', f'<@{mention.id}>', 1)
+
+        return string
+
+
+class Reactions(commands.Cog):
+    def __init__(self, bot: CrimsoBOT):
+        self.bot = bot
+
+    @commands.group(aliases=['facts'])
+    async def fact(self, ctx: commands.Context) -> None:
+        """Testing out facts."""
+        return
+
+    @fact.command(name='about')
+    async def fact_about(self, ctx: commands.Context, *, subject: CleanMentions) -> None:
+        """Write up some HELP"""
+        try:
+            fact_object = await FunFacts.get_by_subject(subject.lower().strip(), ctx.guild.id)
+        except NoFactsExist:
+            embed = c.crimbed(
+                title='OMAN',
+                descr=f'There are no facts about {subject}!',
+                footer='You can add one by using ">fact about [fact subject]; [some fun fact]"',
+                color_name='yellow',
+                thumb_name='weary',
+            )
+            await ctx.send(embed=embed, delete_after=18)
+            return
+
+        return_string = ''.join([
```

Comment:
I feel it's probably cleaner to rewire this section into a try/except/else block, so that you have better readability and don't need to use a return statement.